chaos/cluster/cluster.go

```diff
@@ -84,55 +87,40 @@ func (cl *Cluster) RebootAll(rebootAfter time.Duration) error {
 	}
 
 	glog.V(4).Infof("will reboot nodes: %s", hosts)
-	maxParallel, err := intstr.GetValueFromIntOrPercent(&cl.MaxDisruption, len(hosts), true)
-	if err != nil {
-		return fmt.Errorf("errors parsing max disruption: %v", err)
-	}
-
-	return cl.rebootHosts(hosts, maxParallel, rebootAfter)
+	return cl.rebootHosts(hosts, rebootDuration)
 }
 
-// RebootAllMasters reboots all the master nodes that are accessible ie. have ExternalIP.
-// rebootAfter is the time duration between stopping kubelet and issuing reboot.
-func (cl *Cluster) RebootAllMasters(rebootAfter time.Duration) error {
+// RebootMasters reboots all the master nodes that are accessible ie. have ExternalIP.
+// rebootDuration is the time duration between stopping kubelet and issuing reboot.
```

Comment:
i am not a fine of documenting implementation details. but if we want, we need to be accurate. we also tear down the interface.